### ISSUE TITLE
Example_fix

### ISSUE THREAD

**Issue by Janicevd** [AckslD/QuAlg#20]
I've made a little adjustment to the POVM example (not very prettily, but working)  such that the factor of 1/2 in the projectors P20 and P02 are taken into account, but I can't push. The changes I've made (**bold**) are in construct_projector():

```python
    elif (num_left, num_right) == (2, 0):
        **# P20, which contains a factor of 1/2 s.t. P20^2=P20, this can be added by rescaling c1:
        c1 = BaseFockState([FockOp('c', 'p1')]).to_state()*(1/np.sqrt(2))**
        return outer_product(c1@c2, c1@c2)
    elif (num_left, num_right) == (0, 2):
        **# P02, which contains a factor of 1/2 s.t. P02^2=P02, this can be added by rescaling d1:
        d1 = BaseFockState([FockOp('d', 'p1')]).to_state()*(1/np.sqrt(1/2))**
        return outer_product(d1@d2, d1@d2)
```

**Comment by AckslD** [comment]
Hi @Janicevd! Thanks for pointing this out!

I edited your comment to add a proper code block, you can use triple-quotes for this.

To make a pull request you can fork the project, make your edits and submit the request.

Regarding dividing by a scalar, I added support for that some time ago in #19 but forgot merge. Would you like to review that?

**Comment by AckslD** [comment]
closed by #21 and #19
